locale/inc/leatherman/locale/locale.hpp

```diff
@@ -15,12 +15,11 @@
 #include <locale>
 #include <vector>
 #include <functional>
-
-#ifdef LEATHERMAN_I18N
```

Comment:
I'm not sure why you changed this. Simply including boost::locale on some platforms will be a problem; I don't think we can link against it on one of them (maybe AIX).